include/SDL3/SDL_time.h

```diff
@@ -0,0 +1,338 @@
+ *
+ * %m - The month as a decimal number [01-12].
+ *
+ * %M - The minute as a decimal number [00-59].
+ *
+ * %n - A newline character.
+ *
+ * %p - Either "AM" or "PM" according to the given time value, or the corresponding strings for the
+ *      current locale.  Noon is treated as "PM" and midnight as "AM".
+ *
+ * %P - Like %p but in lowercase: "am" or "pm" or a corresponding string for the current locale.
+ *
+ * %r - The time in a.m. or p.m. notation.
+ *
+ * %R - The time in 24-hour notation (%H:%M).
+ *
+ * %s - The number of seconds since the Epoch, 1970-01-01 00:00:00 +0000 (UTC).
+ *
+ * %S - The second as a decimal number [00-60].
+ *
+ * %t - A tab character.
+ *
+ * %T - The time in 24-hour notation (%H:%M:%S).
+ *
+ * %u - The day of the week as a decimal, range 1 to 7, with Monday being 1.
+ *
+ * %U - The week number of the current year as a decimal number, range 00 to 53, starting with the first
+ *      Sunday as the first day of week 01.
+ *
+ * %V - The ISO 8601 week number of the current year as a decimal number, range 01 to 53, where week 1 is
+ *      the first week, starting on Monday, that has at least 4 days in the new year.
+ *
+ * %w - The day of the week as a decimal, range 0 to 6, with Sunday being 0.
+ *
+ * %W - The week number of the current year as a decimal number, range 00 to 53, starting with the first
+ *      Monday as the first day of week 01.
+ *
+ * %x - The preferred date representation for the current locale without the time.
+ *
+ * %X - The preferred time representation for the current locale without the date.
+ *
+ * %y - The year as a decimal number without a century [00-99].
+ *
+ * %Y - The year as a decimal number including the century.
+ *
+ * %z - The +hhmm or -hhmm numeric timezone (the hour and minute offset from UTC).
+ *
+ * %Z - The timezone name or abbreviation, if available.
+ *
+ * %% - A literal '%' character.
+ *
+ * Some conversion specifications can be modified by preceding the conversion specifier character
+ * by the E or O modifier to indicate that an alternative format should be used. If the alternative
+ * format or specification does not exist for the current locale, the behavior will be as if the
+ * unmodified conversion specification were used. Most POSIX implementations support %Ec, %EC, %Ex,
+ * %EX, %Ey, %EY, %Od, %Oe, %OH, %OI, %Om, %OM, %OS, %Ou, %OU, %OV, %Ow, %OW, %Oy, where the effect of
+ * the O modifier is to use alternative numeric symbols, and that of the E modifier is to use a
+ * locale-dependent alternative representation, such as the Japanese era calendar scheme. This is highly
+ * platform-dependent and the documentation for strftime should be consulted to determine the capabilities
+ * of your specific platform. Note that Windows uses era modifiers of the form '#' instead of 'E'
+ * (e.g. '%#c' instead of '%Ec'), however, SDL requires that the POSIX compliant formatting rules be used.
```

Comment:
I suggest removing this paragraph and adding a shorter YMMV note, e.g.
> Some platforms have additional conversion specifications, but they are not available across all platforms and you should consult your system documentation if you want to use them.

Reply:
Reworded to be much more concise.